### DISCUSSION TITLE
plot arka planda çalışıyor

### DISCUSSION THREAD

**Discussion by mgxmurat** [Chia-Network/chia-blockchain#4673] [Support]
plot yapma arka planda geçici dosyada çalışıyor ama ekranda görünmüyor bu dosyayı silmelimiyim

**Comment by oytp** [comment]
If you want to get fast support, please, use English. Respect other members

**Reply by mgxmurat** [reply]
Good day, I apologize to everyone for not writing in English before. Plotting seems to be finished in the temp file and works in the background, but it does not appear on the chia screen and it did not transfer from m2 to hdd. I need to delete my gateway files. (By the way, the program was closed and opened once.

**Reply by oytp** [reply]
> Good day, I apologize to everyone for not writing in English before. Plotting seems to be finished in the temp file and works in the background, but it does not appear on the chia screen and it did not transfer from m2 to hdd. I need to delete my gateway files. (By the way, the program was closed and opened once.

If you start plotting, wait until it ends after 100%. When you close the process, all the temp files stay on your temp disk (you use m.2 SSD probably), so you have to delete temp files and only then start plotting from the beginning

**Reply by mgxmurat** [reply]
thank you so much

**Comment by mgxmurat** [comment]
When creating a plot, I wonder if there is an advantage of k33 or k34 instead of k32.

**Reply by oytp** [reply]
> When creating a plot, I wonder if there is an advantage of k33 or k34 instead of k32.

Devs say  they expect plotting k33 and bigger after 2026

**Reply by mgxmurat** [reply]
thank you so much

**Reply by mgxmurat** [reply]
I did 33 and 34 for a couple of skins, and after that, thank you again for giving me the idea to keep doing k 32 all the time.

**Comment by mgxmurat** [comment]
While creating the plot, I can create 4 plots at the same time according to the capacity of the computer, then I can create these 4-plots manually each time for later, I can create 4-piece 10 parallel plots. I wonder if this should be done with power_shell

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
